src/util/result.h

```diff
@@ -18,9 +19,9 @@ class BResult {
     std::variant<bilingual_str, T> m_variant;
 
 public:
-    BResult() : m_variant(Untranslated("")) {}
-    BResult(const T& _obj) : m_variant(_obj) {}
```

Comment:
In commit "refactor: Return BResult from restoreWallet" (fa869c6df15a54dae0e8d6ba385514801ddd345f)

I think dropping `const T&` constructor drops support for noncopyable and nonmovable objects.

Maybe want:

```c++
BResult(const T& obj) : m_variant{obj} {}
BResult(T&& obj) : m_variant{std::move(obj)} {}
```

Reply:
> drops support

This was never supported, unless I am missing something?

Regardless, it seems wrong to return memory that is not owned. Though, if you really want to, it might be better to make the type `T` explicit. That is, make it a a raw pointer/std::ref/span/string_view/whatever.